src/uwtools/drivers/ungrib.py

```diff
@@ -113,12 +112,13 @@ def runscript(self):
     @task
     def vtable(self):
         """
-        The Vtable.
+        A symlink to the Vtable file.
         """
         path = self._rundir / "Vtable"
         yield self._taskname(str(path))
         yield asset(path, path.is_symlink)
-        yield None
+        infile = Path(self._driver_config["vtable"])
+        yield file(path=infile)
```

Comment:
This task was previously creating the symlink even if its target did not exist, which isn't what we want.

Before:
```
% uw ungrib vtable --config ungrib.yaml --cycle 2024-03-22T18
[2024-03-22T23:00:15]     INFO Validating config against internal schema ungrib
[2024-03-22T23:00:15]     INFO 0 UW schema-validation errors found
[2024-03-22T23:00:15]     INFO Validating config against internal schema platform
[2024-03-22T23:00:15]     INFO 0 UW schema-validation errors found
[2024-03-22T23:00:15]     INFO 20240322 18Z ungrib /tmp/run/Vtable: Initial state: Pending
[2024-03-22T23:00:15]     INFO 20240322 18Z ungrib /tmp/run/Vtable: Checking requirements
[2024-03-22T23:00:15]     INFO 20240322 18Z ungrib /tmp/run/Vtable: Requirement(s) ready
[2024-03-22T23:00:15]     INFO 20240322 18Z ungrib /tmp/run/Vtable: Executing
[2024-03-22T23:00:15]     INFO 20240322 18Z ungrib /tmp/run/Vtable: Final state: Ready
% ll /tmp/run/Vtable 
lrwxrwxrwx 1 maddenp maddenp 13 Mar 22 23:00 /tmp/run/Vtable -> /no/such/file
```

After:
```
% uw ungrib vtable --config ungrib.yaml --cycle 2024-03-22T18
[2024-03-22T23:01:23]     INFO Validating config against internal schema ungrib
[2024-03-22T23:01:23]     INFO 0 UW schema-validation errors found
[2024-03-22T23:01:23]     INFO Validating config against internal schema platform
[2024-03-22T23:01:23]     INFO 0 UW schema-validation errors found
[2024-03-22T23:01:23]     INFO 20240322 18Z ungrib /tmp/run/Vtable: Initial state: Pending
[2024-03-22T23:01:23]     INFO 20240322 18Z ungrib /tmp/run/Vtable: Checking requirements
[2024-03-22T23:01:23]  WARNING File /no/such/file: State: Pending (EXTERNAL)
[2024-03-22T23:01:23]     INFO 20240322 18Z ungrib /tmp/run/Vtable: Requirement(s) pending
[2024-03-22T23:01:23]  WARNING 20240322 18Z ungrib /tmp/run/Vtable: Final state: Pending
% ll /tmp/run/Vtable 
ls: cannot access '/tmp/run/Vtable': No such file or directory
```